grammars/stata.cson

```diff
@@ -83,7 +83,7 @@ patterns: [
       3: name: 'keyword.control.flow.stata'
   }
   {
-    begin: '\\b(foreach)\\s+((?!in|of).+)\\s+(of loc(al|a)?|of glo(bal|ba|b)?)\\b\\s*'
+    begin: '\\b(foreach)\\s+((?!in |of ).+)\\s+(of loc(al|a)?|of glo(bal|ba|b)?)\\b\\s*'
```

Comment:
Same here

Reply:
Oops, completely missed that there were 2. Fixed, thanks!